samples/DependencyInjection/Simple/Simple.nfproj

```diff
@@ -21,19 +21,18 @@
     <Compile Include="Program.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
-  <ItemGroup>
-    <None Include="packages.config" />
-  </ItemGroup>
   <ItemGroup>
     <Reference Include="mscorlib, Version=1.14.3.0, Culture=neutral, PublicKeyToken=c07d481e9758c731">
       <HintPath>packages\nanoFramework.CoreLibrary.1.14.2\lib\mscorlib.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="nanoFramework.DependencyInjection, Version=1.0.35.0, Culture=neutral, PublicKeyToken=c07d481e9758c731">
-      <HintPath>packages\nanoFramework.DependencyInjection.1.0.35\lib\nanoFramework.DependencyInjection.dll</HintPath>
-      <Private>True</Private>
+    <Reference Include="nanoFramework.DependencyInjection">
```

Comment:
This doesn't look correct... can you please double check?
If the library is not being published with strong name then something is not OK on the project upstream...

Reply:
The upstream libraries getting published with strong name.

The issue appears to be that the sample projects are not being signed (which makes sense) so the NuGet UI does not add the required strong name details.

When the dependency update check runs it will add these details (not sure if that's nanodu or nuget.exe doing it).

Let me see if there is a simple way for me to get these details included in the projects that doesn't involve manually hand editing them all ;)

Reply:
The "solution" is to downgrade the packages in the projects and then use nuget.exe from the command line to upgrade them.

I'll try to create a temp solution that pulls in all these samples to make the process less manual.

That being said do we think this data in the project file actually has any impact on anything? It seems that this data could be pulled from the libraries at build time (not sure that is how it works).

Reply:
OK. Let's go with this.